README.rst

```diff
@@ -30,6 +40,60 @@ Xarray Extended with Climate Data Analysis Tools
 .. |Checked with mypy| image:: http://www.mypy-lang.org/static/mypy_badge.svg
    :target: http://mypy-lang.org/
 
+
+XCDAT is a Python library built on `xarray`_ for climate data analysis on structured grids.
+It serves as a spiritual successor to the Community Data Analysis Tools (`CDAT`_) library with a focus on long-term maintainability and extensibility.
+
+.. _xarray: https://github.com/pydata/xarray
+.. _CDAT: https://github.com/CDAT/cdat
+
+Planned Features
+-----------------
+
+XCDAT will provide climate domain specific features and utilities that are helpful for general xarray usage.
+
+These features include:
+
+- Support for metadata that is CF compliant, E3SM non-CF compliant, and common
+- Robust handling of coordinates and its associated bounds
+
+  - Name-agnostic retrieval of CF compliant coordinates and bounds using ``cf_xarray``
+  - Generating specific bounds or filling all missing bounds for supported axes
+  - Ability to operate on both (0 to 360) and (-180 to 180) longitudinal axes orientations
+
+- Temporal averaging (weighted or unweighted)
+
+  - Time series averaging and calculation of climatologies and anomalies
+  - Use of time bounds for calculating weights
+  - Optional centering of time coordinates using time bounds
+
+- Geospatial weighted averaging over rectilinear grid
+
+  - Optional specification of regional domain
+
+- Horizontal and vertical structured regridding
+
+  - Operate on rectilinear and cuvilinear grids
+
+- Wrappers for opening datasets to apply common operations
+
+  - Fill missing bounds
+  - Decoding of CF and non-CF time units
+  - Optional centering of time coordinate using time bounds
+  - Optional conversion of longitudinal axes orientation
+
+- Support for parallelism of XCDAT features using Dask
+
+Criteria
+~~~~~~~~
+
+The features of this library must meet a set of criteria before being considered for implementation.
+
+1. Climate domain functionality and/or general ``xarray`` utility isn't provided natively with ``xarray``
+2. No existing xarray-based packages implement the feature, or the implementation doesn't meet the XCDAT team's defined requirements
+3. Feature can be relatively simple to implement and not overly-flexible
```

Comment:
If someone wants to contribute code that is really flexible - is that a problem?

Reply:
If an API for a method/function is overly-flexible, it is probably performing too many operations. This makes it much more difficult to unit test all of the conditionals in the callstack (our unit testing enforces simple and maintainable code!).

I don't think it is a problem if we perform thorough reviews in the PRs and suggest changes if necessary so that the contributions are well-scoped.